protocol/src/lib.rs

```diff
@@ -138,14 +135,14 @@ pub enum RevokeReason {
 #[cfg_attr(feature = "bincode", derive(Encode, Decode))]
 pub enum RejectReason {
     AlreadyExists,
-    BidPSBT(BidPsbtReason),
+    BidPsbt(BidPsbtReason),
 }
 
 #[derive(Copy, Clone, Debug, PartialEq, Eq)]
 #[cfg_attr(
     feature = "serde",
     derive(Serialize, Deserialize),
-    serde(rename_all = "snake_case", tag = "bid_psbt_reason")
+    serde(rename_all = "snake_case")
```

Comment:
do you have an example of this looks like as JSON now for revoke and reject?

Reply:
Here are the responses examples
```
{"type":"reject","name":"@artfuldev","reason":"already_exists"}
{"type":"reject","name":"@tap4","reason":{"bid_psbt":"output_spent"}}
```

Reply:
hm so the reason is now either an object or a string previously for revoke it was like this 

```json
{ "revoke_reason": "bid_psbt", "bid_psbt_reason": "output_spent" } 
```

could we make the reject look the same in a simple way e.g.

```json
{"type":"reject","name":"@artfuldev","reason":"already_exists"}
{"type":"reject","name":"@tap4","reason": "bid_psbt", "bid_psbt_reason":"output_spent"}
```

Reply:
Fixed in #30